### DISCUSSION TITLE
The z1 surface of the cylindrical area cannot be generated correctly in the inp file

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#245] [Q&A] [answered]
My version is the latest version. The command is as follows:
```
neper -T -n 500 -domain "cylinder(4,2)" -o test
neper -M test.tess -nset "z0,z1" -faset none -elttype hex -format inp
```
Check the .inp file, you can find that the set z1 has not been created correctly.

[test.txt](https://github.com/rquey/neper/files/6631737/test.txt)

**Comment by rquey** [comment, marked as answer]
What is wrong with the inp file exactly?

**Reply by smallrain1996** [reply]
Thank you very much for your reply.
You can notice that the nodes in the node set "*Nset, nset=z1" are actually not created. There are only 57005 nodes in the inp file, but the node numbers in "*Nset, nset=z1" are 69697 to 70785. There is no way to upload files in inp format. I copied the files into txt and uploaded them. Or you can run the above two lines of commands on your device and import the inp file into abaqus

**Reply by smallrain1996** [reply]
In addition, I recently installed version 4.2.9 and the problem still exists.

**Reply by rquey** [reply]
Converted into issue #312.
